readyplayerme/rpm-unreal-next-gen-sdk
# Feature/dev window update

## [PHO-359](https://ready-player-me.atlassian.net/browse/PHO-359)

-   This PR improves the code for the Developer login Editor window, it was previously 1 giant class that controlled 2 separate panel views. But now these separate views (or panels) are in their own self contained classes RpmDeveloperLoginPanel & RpmDeveloperSettingsPanel. The RpmDeveloperLoginWidget is still there and it manages the switching between the two amony other things. 
- added error messages to the developer widget panels 

![image](https://github.com/user-attachments/assets/c3eb7669-3c86-4643-8ea8-6faba3aad451)
![image](https://github.com/user-attachments/assets/8f80a1d5-93ed-4049-8887-12e0bc66d5ca)

- login button disabled if email or password are empty 






## How to Test

-   Test all previous functionality still works 
        - Login with email and password
        - login with demo account
        - logout
        - import base model from import button
        - application dropdown should auto populate
        - If you have logged in previously it should stay logged in for next time 



## Checklist


## QA Testing

-   Mention any useful test cases for this feature and add to QA Testing documentation.

## Details

-   If more details are necessary to support the description with images and videos add them here.







[PHO-359]: https://ready-player-me.atlassian.net/browse/PHO-359?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
